### PR TITLE
Nexus 21943 Conan search support

### PR DESCRIPTION
**Pull request by igorudovika**
Nexus 21943 Conan search support

### CODE REVIEW

File: nexus-repository-conan/src/main/resources/static/rapture/NX/conan/controller/SerachConan.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      text: NX.I18n.get('SearchConan_Text'),
+      description: NX.I18n.get('SearchConan_Description'),
+      readOnly: true,
+      criterias: [
+        {id: 'format', value: 'conan', hidden: true},
+        {id: 'name.raw'}
+      ]
+    }, me);
+  }
+});
```
Comment (DreamVan) [inline nexus-repository-conan/src/main/resources/static/rapture/NX/conan/controller/SerachConan.js]: Add last empty line

File: nexus-repository-conan/src/main/resources/static/rapture/NX/conan/controller/SerachConan.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+   */
+  init: function() {
+    var me = this,
+        search = me.getController('NX.coreui.controller.Search');
+
+    search.registerFilter({
+      id: 'conan',
+      name: 'Conan',
+      text: NX.I18n.get('SearchConan_Text'),
+      description: NX.I18n.get('SearchConan_Description'),
```
Comment (DreamVan) [inline nexus-repository-conan/src/main/resources/static/rapture/NX/conan/controller/SerachConan.js]: Question: I don't see changes for PluginStrings. Was it added previously?

Comment (igorudovika) [reply nexus-repository-conan/src/main/resources/static/rapture/NX/conan/controller/SerachConan.js]: Yes, such JS code is already present

File: nexus-repository-conan/src/main/resources/static/rapture/NX/conan/controller/SerachConan.js (hunk 1 above)
Comment (igorudovika) [reply nexus-repository-conan/src/main/resources/static/rapture/NX/conan/controller/SerachConan.js]: Fixed e977318b7d4a644b4b75adfae4a2dcd653a9b054

**Pull request merged**